examples/X11/X11.cpp

```diff
@@ -10,18 +10,24 @@
 #include <X11/Xlib.h>
 #include <iostream>
 #include <cmath>
-
+#include <cstdlib>
 
 ////////////////////////////////////////////////////////////
 /// Initialize OpenGL states into the specified view
 ///
 /// \param Window Target window to initialize
 ///
+/// \return True if operation was successful, false otherwise
+///
 ////////////////////////////////////////////////////////////
-void initialize(sf::Window& window)
+[[nodiscard]] bool initialize(sf::Window& window)
 {
     // Activate the window
-    window.setActive();
+    if (!window.setActive())
+    {
+        std::cerr << "Failed to set window to active" << std::endl;
```

Comment:
Should this be `sf::err()`?

(applies to others too)

Reply:
I have always assumed that `sf::err` is SFML's *internal* error stream. Isn't that the case?

Reply:
Oh, I missed that this is an example, you're right.
Only the Vulkan.cpp example uses `sf::err()`, and that's probably by accident.